### DISCUSSION TITLE
Regarding FDS for solving zero dimensional pyrolysis problem

### DISCUSSION THREAD

**Discussion by RimB2301** [firemodels/fds#13139] [Q&A]
![error_fds](https://github.com/firemodels/fds/assets/175034264/d125bd25-d6af-425d-96c5-766381a8cdcb)
Hi, I am Ayona, a postgraduate student in the Mechanical Engineering Department at Imperial College London. I tried running the FDS code to solve a zero-dimensional pyrolysis problem for PMMA. However, I encountered an error (picture attached). Can anybody please help me fix this error?

Thank you!

**Comment by drjfloyd** [comment]
We would need to see your input file.

**Reply by RimB2301** [reply]
Sure, I am attaching the input file here. 

Thank you!

Code: 

&HEAD CHID='foam_tga', TITLE='TGA test of foam at a heating rate of 10 K/min' / 

&MESH IJK=3,1,4, XB=-2,2,-0.5,0.5,0,1 / 

&TIME T_END=60. /

&REAC FUEL='CELLULOSE', C=6, H=10, O=5, SOOT_YIELD=0.015 /

&VENT XB=-1,1,-0.5,0.5,0.0,0.0, SURF_ID='SAMPLE' /

&SURF ID             = 'SAMPLE'
      TGA_ANALYSIS   = T
      TGA_HEATING_RATE = 10. !10 K/min
      COLOR          = 'RED' 
      THICKNESS      = 0.01
      MATL_ID(1,1) = 'FOAM'
      MATL_ID(1,2) = 'BETA_FOAM' 
      MATL_MASS_FRACTION(1,1) = 1.
      MATL_MASS_FRACTION(1,2) = 0. /

&MATL ID                    = 'FOAM'
      EMISSIVITY            = 0.9
      DENSITY 	            = 1000.
      CONDUCTIVITY          = 1.
      SPECIFIC_HEAT         = 1.
      N_REACTIONS           = 1
      NU_SPEC 	            = 0.8
      A                     = 2.0E12
      E                     = 148.0
      HEAT_OF_REACTION      = 500.0 
      NU_MATL               = 0.7
      MATL_ID               = 'BETA_FOAM'
      NU_SPEC               = 0.3
      SPEC_ID               = 'CELLULOSE' /  

&MATL ID                    = 'CHAR'
      EMISSIVITY            = 1.0
      DENSITY 	            = 35.
      CONDUCTIVITY          = 0.40
      SPECIFIC_HEAT         = 1.0  /

&MATL ID                    = 'BETA_FOAM'
      DENSITY               = 700.
      EMISSIVITY            = 0.9
      CONDUCTIVITY          = 0.1
      SPECIFIC_HEAT         = 4.184
      N_REACTIONS           = 1
      NU_SPEC               = 1.0
      A                     = 1.6E9
      E                     = 124.0
      HEAT_OF_REACTION      =500.0 
      NU_MATL               = 0.05
      MATL_ID               = 'CHAR'
      NU_SPEC               = 0.95
      SPEC_ID               = 'CELLULOSE' /

&TAIL /

**Reply by drjfloyd** [reply]
Are you using the current release?  This case runs for me with 6.9.1

```
 MPI Process      0 started on L-PW078R66

 Reading FDS input file ...

WARNING: SPEC CELLULOSE is not in the table of pre-defined species. Any unassigned SPEC variables in the input were assigned the properties of nitrogen.

 Fire Dynamics Simulator

 Current Date     : July  8, 2024  12:09:36
 Revision         : FDS-6.9.1-0-g889da6a-release
 Revision Date    : Sun Apr 7 17:05:06 2024 -0400
 Compiler         : Intel(R) Fortran Intel(R) 64 Compiler Classic for applications running on Intel(R) 64, Version 2021.9.0 Build 20230302_000000
 Compilation Date : Mon 04/08/2024  10:32 PM

 Number of MPI Processes:  1

 MPI version: 3.1
 MPI library version: Intel(R) MPI Library 2021.10 for Windows* OS


 Job TITLE        : TGA test of foam at a heating rate of 10 K/min
 Job ID string    : foam_tga


STOP: FDS performed a TGA analysis only and finished successfully (CHID: foam_tga)
```

**Reply by drjfloyd** [reply]
make sure you are following the instructions on running FDS with windows.

**Comment by RimB2301** [comment]
Yes, I used the same version. Could you please suggest what else could be done?
![fds_version](https://github.com/firemodels/fds/assets/175034264/b2cff228-d0e2-4730-96da-1eb2b6075252)

**Comment by drjfloyd** [comment]
Don't run in OneDrive folder. That can cause problems if a file is synching when FDS is trying to access the folder.

The instructions in the User's Guide for windows are to run using the FDS CMD shell and fds_local. You do not appear to be doing either.

**Comment by RimB2301** [comment]
Yes, I  changed the file location and tried to run the code again, but I found this error. 
![fds_error](https://github.com/firemodels/fds/assets/175034264/f0eee4b9-980e-4b8e-85b5-d0656a266e42)

**Reply by drjfloyd** [reply]
Are you using the CMDfds shortcut on your desktop before trying to run FDS?

**Comment by RimB2301** [comment]
Thank you so much for your help. I fixed the error now. 
![error_fix](https://github.com/firemodels/fds/assets/175034264/b0b2facc-f509-413c-9cab-d624799397c7)
